src/sparseml/transformers/text_classification.py

```diff
@@ -630,6 +649,11 @@ def compute_metrics(p: EvalPrediction):
     else:
         data_collator = None
 
+    # Log a few random samples from the training set:
+    if training_args.do_train:
+        for index in random.sample(range(len(train_dataset)), 3):
```

Comment:
what's the reason for this?

Reply:
sanity check to make sure that we are feeding the correct data structure into the model? if my assumption is correct, it would be probably more useful for the user to see something like 
` _LOGGER.info(f"Sample {index} of training set: {train_dataset[index].size()}.")`
rather than full tensor print.